c_glib/arrow-glib/type.h

```diff
@@ -50,6 +50,8 @@ G_BEGIN_DECLS
  * @GARROW_TYPE_TIME64: Exact time encoded with int64, supporting micro- or nanoseconds
  * @GARROW_TYPE_MONTH_INTERVAL: YEAR_MONTH interval in SQL style.
  * @GARROW_TYPE_DAY_TIME_INTERVAL: DAY_TIME interval in SQL style.
+ * @GARROW_TYPE_DECIMAL64: Precision- and scale-based decimal
+ *   type with 64-bit. Storage type depends on the parameters.
```

Comment:
Could you use the same order as the definition? (Could you move this to after the `@GARROW_TYPE_RUN_END_ENCODED`?)

Reply:
I changed the order in 8217b31